frontend/web/components/modals/CreateRole.tsx

```diff
@@ -462,6 +462,7 @@ const CreateRole: FC<CreateRoleType> = ({
                   ))}
               </div>
             </TabItem>
+            {/* add padding:0*/}
```

Comment:
Not sure why this is here?

Reply:
It was a comment to fix a design style, fixed and the comment deleted.